src/page-multi-facility/MultiFacilityPage.tsx

```diff
@@ -27,7 +51,17 @@ const MultiFacilityPage: React.FC = () => {
             <div className="flex flex-col flex-1 pb-6 pl-8">
               <AddFacilityModal />
               <ProjectionsHeader />
-              <div>Projections will go here</div>
+              {facilities.loading ? (
+                <Loading />
+              ) : (
+                facilities.data.map((facility, index) => {
```

Comment:
there's still the possibility that `facilities.data` will be null here .... I recommend using `facilities.data?.map` as a guard against that (it should just return null instead of proceeding to try and call `.map()`)

Reply:
Good catch!  Fixed it locally.  We'll squash my commits and merge now.